keyboards/peej/lumberjack/lumberjack.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This is a VIA json and as such doesn't belong in this repo.  This needs to PRed to the the-via/keyboards repo.